keyvault/client/keys/src/main/java/com/azure/keyvault/keys/KeyAsyncClient.java

```diff
@@ -56,6 +56,8 @@
     static final String ACCEPT_LANGUAGE = "en-US";
     static final int DEFAULT_MAX_PAGE_RESULTS = 25;
     static final String CONTENT_TYPE_HEADER_VALUE = "application/json";
+    static final String KEY_VAULT_SCOPE = "https://vault.azure.net/.default";
```

Comment:
I wasn't sure where all the KV constants go. So I duplicated it in 4 places similar to all the other clients I saw. I'm sure there is a better place for this.